### DISCUSSION TITLE
Global setting for disabling GitHub Issues

### DISCUSSION THREAD

**Discussion by pascal-hofmann** [renovatebot/renovate#30050] [Suggest an Idea]
### Tell us more.

Hi,
we have GitHub Issues disabled for all repositories in our organization. It would be great if there was a way to disable creating GitHub Issues globally. Currently this just generates a lot of log messages:
> Cannot ensure issue because issues are disabled in this repository

Cheers
Pascal

**Comment by rarkins** [comment]
What type of issue is causing this?

If it's dashboard, you can turn that off. 

If it's a config error, you should want to get a log message and then fix it so it goes away.

**Reply by pascal-hofmann** [reply]
Hi Rhys,
thanks for you quick response!

The dashboard is disabled. This causes no issue apart from ~5000 log messages per day that make it a bit harder to spot real issues.

(I also noticed that this message is printed multiple times per repo. I guess once is enough, even in case of a config error.)

Cheers
Pascal

**Reply by rarkins** [reply]
If you have the dashboard disabled then I'm not sure why you have so many not messages. I was asking you to work out the reason why.

**Reply by rarkins** [reply]
The warning message you are concerned with should include the title in the log metadata

**Reply by pascal-hofmann** [reply]
I could see more details when running with log level debug:
```
{"name":"renovate","hostname":"6d362b5750ed","pid":10,"level":10,"logContext":"mVcLBi68V0F-3hywzbOKW","repository":"redacted/redacted-renovate-testing","msg":"ensureIssueClosing(Action Required: Fix Renovate Configuration)","time":"2024-07-05T09:16:46.115Z","v":0}
{"name":"renovate","hostname":"6d362b5750ed","pid":10,"level":30,"logContext":"mVcLBi68V0F-3hywzbOKW","repository":"redacted/redacted-renovate-testing","msg":"Cannot ensure issue because issues are disabled in this repository","time":"2024-07-05T09:16:46.115Z","v":0}
{"name":"renovate","hostname":"6d362b5750ed","pid":10,"level":10,"logContext":"mVcLBi68V0F-3hywzbOKW","repository":"redacted/redacted-renovate-testing","msg":"ensureIssueClosing(Action Required: Add missing credentials)","time":"2024-07-05T09:16:46.115Z","v":0}
{"name":"renovate","hostname":"6d362b5750ed","pid":10,"level":30,"logContext":"mVcLBi68V0F-3hywzbOKW","repository":"redacted/redacted-renovate-testing","msg":"Cannot ensure issue because issues are disabled in this repository","time":"2024-07-05T09:16:46.115Z","v":0}
```

**Reply by rarkins** [reply]
You have a config problem. Fix that and you shouldn't have any issues noise.

**Reply by pascal-hofmann** [reply]
I can't see any issue with the repo. This code is run for all repos: 
https://github.com/renovatebot/renovate/blob/3e3bb63751bea8d4539e272604e128364d5d9ded/lib/workers/repository/finalize/index.ts#L47

**Reply by pascal-hofmann** [reply]
I guess you mistook this for https://github.com/renovatebot/renovate/blob/3e3bb63751bea8d4539e272604e128364d5d9ded/lib/workers/repository/error-config.ts#L18 and https://github.com/renovatebot/renovate/blob/3e3bb63751bea8d4539e272604e128364d5d9ded/lib/workers/repository/error-config.ts#L29 but both functions are not involved.

**Reply by rarkins** [reply]
Yes, I missed that it's closing. We should gracefully handle it in that case

**Comment by rarkins** [comment]
Fix PR:

- https://github.com/renovatebot/renovate/pull/30267
